Source/Engine/PackageBuilder/SyntaxParser.cs

```diff
@@ -96,9 +97,13 @@ public PackageSyntax ParseExpressionText(string text)
             fTextPosition = -1;
             NextCharacter();
             NextToken();
+            int startPosition = fToken.TextSlice.Position;
             Syntax patternBody = ParseInsideOrOutsideOrHaving();
+            AttachSourceTextInformation(patternBody, startPosition);
```

Comment:
`AttachSourceTextInformation` was already called in `ParseInsideOrOutsideOrHaving`